src/pyrokinetics/gk_code/gs2.py

```diff
@@ -893,7 +893,7 @@ def set(
             self.data["normalisations_knobs"]["vref"] = (1 * convention.vref).to(
                 "meter/second"
             )
-            self.data["normalisations_knobs"]["qref"] = 1 * convention.qref
```

Comment:
Should we also convert to Coulombs as we do for all other normalisations?

Reply:
If it should be then yes. I'm not sure what GS2 expects so it should be whatever that is

Reply:
GS2 doesn't do anything with this figure other than writing it out to its netcdf output -- it's primarily to make it easier to keep track of the reference values used in generating the input file.

Reply:
I guess then to be consistent with the other references we should put it in Coulombs